src/app/models/data/Annotation.ts

```diff
@@ -0,0 +1,49 @@
+import { annotationMenuItem } from "@components/library/library.menus";
+import { listenRecordingMenuItem } from "@components/listen/listen.menus";
+import { DateTimeTimezone } from "@interfaces/apiInterfaces";
+import { AbstractModelWithoutId } from "@models/AbstractModel";
+import { IAudioEvent } from "@models/AudioEvent";
+import { AudioRecording } from "@models/AudioRecording";
+import { ITag } from "@models/Tag";
+import { ITagging, Tagging } from "@models/Tagging";
+
+export interface IAnnotation extends Required<IAudioEvent> {
+  tags: ITag[];
+  audioRecording: AudioRecording;
+  audioLink: string;
+}
+
+export class Annotation extends AbstractModelWithoutId implements IAnnotation {
```

Comment:
explain context behind this class

viewmodel

multiple models included to make rendering easier

multiple services used in a meta service to retrieve and create one of these